spec/requests/list_controller_spec.rb

```diff
@@ -5,7 +5,10 @@
 
 describe ListController do
 
-  before { SiteSetting.assign_enabled = true }
+  before do
+    SiteSetting.assign_enabled = true
+    SearchIndexer.enable
```

Comment:
The reason that we disable search indexing by default is that it is quite slow. Can you move this so that it is only enabled in the `filtering of topics` context? That way, it won't make the other tests slower

Reply:
Sure ill do that right now😊